tools/github_webhook_processor.php

```diff
@@ -247,7 +252,7 @@ function checkchangelog($payload, $merge = false) {
 		}
 	}
 	
-	if (!count($changelogbody))
+	if (!count($changelogbody) || $target_branch_ref != "master")
```

Comment:
I suppose, but in this instance it's still only changing one variable's contents (well, you're changing just a literal here but butts I'm just making a point) but I suppose that's easier to locate given it's at the top of the file, so I'll go ahead and do it.